engine/menus/pokedex.asm

```diff
@@ -105,8 +105,8 @@ HandlePokedexSideMenu:
 	jr z, .choseCry
 	dec a
 	jr z, .choseArea
-.choseQuit
-	ld b, 1
+
+	ld b, 1 ; choseQuit
```

Comment:
Was this because `.choseQuit` is unreferenced? The usual convention is to use a comment there instead, e.g. `; quit` or `; choseQuit` or `; chose quit` in place of `.choseQuit`, but still unindented like a label.